symfony/mailer/4.3/manifest.json

```diff
@@ -5,5 +5,14 @@
     "env": {
         "#1": "MAILER_DSN=smtp://localhost"
     },
+    "docker-compose": {
+        "docker-compose.override.yml": {
+            "services": [
```

Comment:
It would be nice to move this to a separate file to instead of writing (maybe more complex) services in json 

This is more a feature request/idea for flex.

WDYT?